src/DocTests.jl

```diff
@@ -190,8 +190,20 @@ function doctest(ctx::DocTestContext, block_immutable::MarkdownAST.CodeBlock)
         end
         if occursin(r"^julia> "m, block.code)
             eval_repl(block, sandbox, ctx.meta, ctx.doc, ctx.file)
-        elseif occursin(r"^# output$"m, block.code)
+        elseif occursin(r"^# output\s*$"m, block.code)
```

Comment:
I think we need to take this away, otherwise the next `if` never triggers?

```suggestion
        elseif occursin(r"^# output$"m, block.code)
```